local.settings.json

```diff
@@ -1,17 +0,0 @@
-{
-    "IsEncrypted": false,
-    "Values": {
-        "AzureWebJobsStorage": "DefaultEndpointsProtocol=https;AccountName=storageaccountfadita8fd;AccountKey=jRbnd+jsfIBnsa+EILtQSMPoilmgJSnsUxBxwj6WbWZINx6tFuwleoutOdAU5h6M4fTvuxTmRzVvTs3hvqunfw==;EndpointSuffix=core.windows.net",
```

Comment:
Make sure you go on and delete/roll the secrets all these compromised resources

Reply:
@faditawfik21

Reply:
Sorry about that, I was running some tests and forgot to delete the local.settings.json file.